### ISSUE TITLE
Can't parse enums with Display attributes

### ISSUE THREAD

**Issue by Tordensmeden** [ffMathy/typescript-cs-poco#60]
This enum fails parsing:

```
    public enum Foobar
    {
        [Display(Name = "Foobar lol")]
        BlahOption,
        [Display(Name = "Blah blah")]
        BlahBlahOption,
        [Display(Name = "Buz bar")]
        BuzBlahOption
    }
```

**Comment by ffMathy** [comment]
This should be fixed now.

**Comment by ffMathy** [comment]
Nope, not fixed.

**Comment by ffMathy** [comment]
Fixed "again" in 4d5c0ca2b0c65747f128c4622f5a723cc45dfdbc.
